cmd/skywire-cli/commands/config/gen.go

```diff
@@ -4,55 +4,134 @@ package cliconfig
 import (
 	"encoding/json"
 	"fmt"
+	"io"
+	"net"
+	"net/http"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 	"strings"
+	"time"
 
 	"github.com/bitfield/script"
-	"github.com/sirupsen/logrus"
+	"github.com/skycoin/dmsg/pkg/disc"
+	"github.com/skycoin/dmsg/pkg/dmsgpty"
+	coinCipher "github.com/skycoin/skycoin/src/cipher"
 	"github.com/spf13/cobra"
 
+	"github.com/skycoin/skywire-utilities/pkg/buildinfo"
 	"github.com/skycoin/skywire-utilities/pkg/cipher"
-	"github.com/skycoin/skywire-utilities/pkg/logging"
 	"github.com/skycoin/skywire-utilities/pkg/netutil"
+	utilenv "github.com/skycoin/skywire-utilities/pkg/skyenv"
 	"github.com/skycoin/skywire/pkg/app/appserver"
+	"github.com/skycoin/skywire/pkg/dmsgc"
+	"github.com/skycoin/skywire/pkg/restart"
 	"github.com/skycoin/skywire/pkg/routing"
 	"github.com/skycoin/skywire/pkg/skyenv"
+	"github.com/skycoin/skywire/pkg/transport/network"
 	"github.com/skycoin/skywire/pkg/visor/visorconfig"
 )
 
+// RootCmd contains commands that interact with the config of local skywire-visor
+var checkPKCmd = &cobra.Command{
+	Use:   "check-pk <public-key>",
+	Short: "check a skywire public key",
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Can we add a check to throw an error if there are more than one keys in the args?

Reply:
a single argument is now enforced